examples/binaries/new-meta/state-v2/src/wasm.rs

```diff
@@ -3,14 +3,16 @@ use gmeta::metawasm;
 use gstd::prelude::*;
 
 #[metawasm]
-pub trait Metawasm {
-    type State = Vec<Wallet>;
+pub mod metafuncs {
+    pub type State = Vec<Wallet>;
 
-    fn wallet_by_id(id: Id, state: Self::State) -> Option<Wallet> {
+    /// Returns a wallet with the given `id`.
+    pub fn wallet_by_id(state: State, id: Id) -> Option<Wallet> {
         state.into_iter().find(|w| w.id == id)
     }
 
-    fn wallet_by_person(person: Person, state: Self::State) -> Option<Wallet> {
+    /// Returns a wallet of the given `person`.
+    pub fn wallet_by_person(state: State, person: Person) -> Option<Wallet> {
```

Comment:
how about parsing Person argument by field directly in fn args?
and also demonstrate more than 1 user arguments parsed